src/Components/StakeTokensModal/__snapshots__/StakeTokensModal.test.tsx.snap

```diff
@@ -0,0 +1,8 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`StakeTokensModal StakeTokensModal renders properly 1`] = `
+<div>
```

Comment:
Hey! Your snapshot isn't working. Since this is a modal and is being created as  `React.createPortal` from document.body you need to add `{container: document.body }` options to the render method in test file. 
Check `src/Components/ActionsModal/ActionsModal.test.tsx` for reference. 
Since this is not a good practice you will have a console.error in your test so you can mock it.

Reply:
Thanks! I totally missed that in the whirlwind of changes. Will fix! And also good to know - had a similar issue with `ProposalVoteCard` and decided to mock it by using `screen` instead.